lib/Distribution/Sle/16Latest.pm

```diff
@@ -0,0 +1,48 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Agama Devel distribution (not really a real distribution!)
+# for integration tests using the current state of code at GitHub for Agama.
+
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Distribution::Sle::16Latest;
+use strict;
+use warnings FATAL => 'all';
+use parent 'susedistribution';
```

Comment:
ci detects duplicate code, what we can do for now is to inherit from devel here so you don't need to rewrite all.

Reply:
Updated.